### DISCUSSION TITLE
Want a FAQ entry to point people at for latest update questions

### DISCUSSION THREAD

**Discussion by rincebrain** [openzfs/zfs#11352] [Ideas]
### Describe the feature would like to see added to OpenZFS

* OpenZFS 2.0.0 (or Linux 5.10) comes out
* Immediate flood of people wondering when they can expect OZFS 2.0 on their platform

It would be useful to have a brief summary package of contact information for the relevant devs, some kind of timeline estimates, and an explanation of how they are already aware that you want the new version, sending them a message just to ask about it probably won't speed it up any.

### How will this feature improve OpenZFS?

Regular streams of people joining IRC and the bug tracker asking the same 3 questions about CentOS 8.3/Streams and Linux 5.10. Sticky bugs explaining these issues would probably be helpful but we could have a more general set of answers available in the FAQ to direct people at, e.g. for "how long will it probably take 0.8.6 versus 2.0.0 to make it into Debian stable distros, and what will that mean for our users).

Ideally, I'd like to try and teach users who come to report "5.11-rc1 isn't working yet do you know" or "why isn't OpenZFS 2.0 packaged on Ubuntu LTS" to check the FAQ or the stickied issues list for answers, rather than opening more.

I'll happily write up explanations for this purpose assuming people would find it useful, and it's not just me overoptimizing a problem, I just wanted to see if other people agree it would be a reasonable way to help users find answers to their common questions and reduce flooding on the bugtracker.

**Comment by behlendorf** [comment]
Good idea.  I'm sure summarizing these answers in an easy to find location, like the [FAQ](https://openzfs.github.io/openzfs-docs/Project%20and%20Community/FAQ.html) would be helpful to many.  Perhaps you could write up what you have in mind and open a PR against the documentation at https://github.com/openzfs/openzfs-docs.

Alternately, (or in addition to) we could enable [GitHub Discussions](https://docs.github.com/en/free-pro-team@latest/discussions) for the OpenZFS repository.  This would provide the project an easily discoverable place to pin announcements, answer common questions, and discuss new ideas/features. It would overlap some with how the mailing list is currently used, but it would probably be more approachable and do a better job keeping questions out of the issue tracker.

**Comment by rincebrain** [comment]
Github Discussions would indeed be a useful way to pin things like "Where's my RH Streams support?" rather than burying it in a FAQ and then linking people there whenever they ask.

I've got a WIP against openzfs-docs; I'll open a PR when I've got more than two examples in it so it illustrates the kind of information I'm hoping to communicate effectively to people who come asking why e.g. OpenZFS 2.0 isn't on Ubuntu on day 1.

[a comment by an automated account is not shown]

**Comment by behlendorf** [comment]
Sounds good.  I've enabled the Discussions tab and started by migrating this issue to the "Ideas" category.  I'll migrate the other issues labeled 'question' to the "Q/A" category, and the 'feature requests' to "Ideas".

[a comment by an automated account is not shown]
